ts/defs.bzl

```diff
@@ -406,6 +416,14 @@ def ts_project(
             visibility = common_kwargs.get("visibility"),
         )
 
+        build_test(
```

Comment:
I'm kind of tempted to not expose this for now, at least as an initial poc.

Reply:
I think this means `test //...` will run this which I think we want to avoid, so we should either drop it for now or tag it as manual...

Reply:
Oh good call